lona/html/data_binding/select.py

```diff
@@ -1,106 +1,44 @@
-from copy import copy
-
-from lona.html.nodes import SelectNode, OptionNode, Widget
 from lona.events.event_types import CHANGE
+from lona.html.node import Node
 
 
-class Select(Widget):
-    ID_LIST = []
-    CLASS_LIST = []
-    STYLE = {}
-    ATTRIBUTES = {}
+class Option(Node):
+    TAG_NAME = 'option'
 
-    def __init__(self, values=[], disabled=False, bubble_up=False,
-                 **select_kwargs):
 
-        self.bubble_up = bubble_up
+class Select(Node):
+    TAG_NAME = 'select'
+    EVENTS = [CHANGE]
 
-        self.select_node = SelectNode(**self.gen_node_args(**select_kwargs))
-        self.select_node.events.add(CHANGE)
+    def __init__(self, *args, values=[], disabled=False, bubble_up=False,
```

Comment:
I didn't tried, but seems following will not work as expected
```py
HTML("<select><option value='a'>a</option><option value='b'>b</option></select>")
```
```py
HTML(Select(Option("a", value="a"), Option("b", value="b")))
```

Reply:
Oh! You are right! The ``Select`` node relies on getting its values as a list of strings, not ``Option`` objects. I will fix that

Reply:
@maratori: Fixed!